### PR TITLE
SQL: group by entity tag #3795

### PR DESCRIPTION
**Pull request by RawrLion**
added tests for #3795

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/clause/groupby/GroupByEntityTag.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        for (int i = 0; i < tags.length; i++) {
+            String testEntityNameTagsCase = entity();
+            Entity entity = new Entity(testEntityNameTagsCase);
+            entity.addTag("tagname", tags[i]);
+            EntityMethod.createOrReplaceEntityCheck(entity);
+
+            Series series = new Series();
+            series.setEntity(testEntityNameTagsCase);
+            series.setMetric(TEST_METRIC_NAME);
+            series.addData(new Sample(Util.ISOFormat(1485525209086L + i), Integer.toString(i)));
```
Comment (VeselovAlex) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/groupby/GroupByEntityTag.java]: > 1485525209086L

OMG

File: src/test/java/com/axibase/tsd/api/method/sql/clause/groupby/GroupByEntityTag.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            series.addData(new Sample(Util.ISOFormat(1485525289086L + i), Integer.toString(i + 1)));
```
Comment (VeselovAlex) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/groupby/GroupByEntityTag.java]: > Integer.toString(i + 1)

May be `BigDecimal.valueOf(i + 1)`?

Comment (vtols) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/groupby/GroupByEntityTag.java]: Tere's `Mocks.ISO_TIME`, is it usable in this case?

File: src/test/java/com/axibase/tsd/api/method/sql/clause/groupby/GroupByEntityTag.java (hunk 1 above)
Comment (vtols) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/groupby/GroupByEntityTag.java]: Tere's `Mocks.ISO_TIME`, is it usable in this case?

Comment (VeselovAlex) [inline src/test/java/com/axibase/tsd/api/method/sql/clause/groupby/GroupByEntityTag.java]: Hardly, cause it is a `String` and @RawrLion wants to use different timestamps

**Pull request merged**